### PR TITLE
feat: add k8s snippet dice yaml intor

### PR DESCRIPTION
**Pull request by iutx**
@sixther-dc @huangzx618

### CODE REVIEW

File: docs/next/manual/dop/guides/reference/dice-yaml.md [hunk 1]
```diff
@@ -321,6 +321,19 @@ health_check:
     duration: 120
 ```
 
+#### k8s_snippet
+配置 service 对 Kubernetes manifest 的 Patch 片段，在一些特殊场景下可能需要使用。
```
Comment (huangzx618) [inline docs/next/manual/dop/guides/reference/dice-yaml.md:325]: The specific adaptation scenario should be explained clearly。

File: docs/next/manual/dop/guides/reference/dice-yaml.md [hunk 2]
```diff
@@ -321,6 +321,19 @@ health_check:
...
+
+示例如下：
+
+修改服务镜像的拉取规则
```
Comment (huangzx618) [inline docs/next/manual/dop/guides/reference/dice-yaml.md:329]: for example：修改服务镜像拉取规则（每次启动都会重新拉取）

Comment (iutx) [reply docs/next/manual/dop/guides/reference/dice-yaml.md]: `Always` is the official kubernetes standard key, just a case, no need to elaborate.

**Review by huangzx618 — APPROVED** [review]
approve

**Pull request merged**